packages/owl-bot/src/database.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import admin from 'firebase-admin';
+import {OwlBotLock} from './config-files';
+import {Configs, ConfigsStore} from './configs-store';
+
+export type Db = admin.firestore.Firestore;
```

Comment:
I honestly have no idea which of these packages (this vs @google-cloud/firestore) we're supposed to use

Reply:
I followed the instructiosn here:
https://firebase.google.com/docs/firestore/quickstart